README.md

```diff
@@ -21,77 +21,44 @@ After responding to changes, merge in GitHub.
 
 ## Setup
 
-### API
-
-* Install JDK 8 on your machine from http://www.oracle.com/technetwork/java/javase/downloads/index.html.
-* Set your JAVA_HOME environment variable to point at the installation directory.
+Docker must be installed to build and run code.
 
 ### UI
 
 * Direct your editor to write swap files outside the source tree, so Webpack
 does not reload when they're updated.
 [Example for vim](https://github.com/angular/angular-cli/issues/4593).
-* Install npm. (For Google workstations, see go/nodejs.)
-
-### General
-
-* Install Google Cloud SDK on your device from https://cloud.google.com/sdk/downloads.
-* If not selected initially, add the AppEngine component with `gcloud components install app-engine-java`.
-* Run tools/setup_env.sh. (This installs angular-cli globally. You may also get it from `ui/node_modules` after
-running `npm install`.)
 
 ## Running the Dev Servers
 
 ### API: dev AppEngine appserver
 
 From the `api/` directory:
-
 ```Shell
-./gradlew appengineRun
+./project.rb dev_up
```

Comment:
I screwed up the deployment, so it isn't connecting to the MySQL database correctly when deployed, but the local spin-up should be working perfectly right now. Brian and I have both tested it, but it would be awesome to have a Chromebook test.